Webapp/sources/models/reaction_note.py

```diff
@@ -18,3 +18,10 @@ class ReactionNote(Model):
         db.ForeignKey("Reaction.id", ondelete="CASCADE"), nullable=False, index=True
     )
     Author = db.relationship("Person", backref="ReactionNote")
+
+    def to_export_dict(self) -> Dict:
```

Comment:
:seedling: I couldn't find where this was being used (if it is yet). 
But it looks like it's serialising data, you might want to look at the `flask-marshmallow` objects in `sources/dto/ReactionNoteSchema`, and create a new one / change etc. 

I'm not against serialising in the models though tbh, as the DTO serialiser approach is non-obvious to Flask.

Reply:
I'll have a look at this in a later related PR since this feature isn't actually implemented yet. I realised I used to_dict() rather than to_export_dict() in the services/data_export.py